api/src/controllers/referent.js

```diff
@@ -864,4 +865,78 @@ router.get("/:id/session-phase1", passport.authenticate("referent", { session: f
   }
 });
 
+router.put("/young/:id/phase1Status/:document", passport.authenticate("referent", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const keys = ["cohesionStayMedical", "autoTestPCR", "imageRight", "rules"];
+    const { error: documentError, value: document } = Joi.string()
+      .required()
+      .valid(...keys)
+      .validate(req.params.document);
+    if (documentError) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS });
+
+    const young = await YoungModel.findById(req.params.id);
+    if (!young) return res.status(404).send({ ok: false, code: ERRORS.NOT_FOUND });
+
+    let values;
+    if (document !== "cohesionStayMedical") {
```

Comment:
plutot faire une whitelist non ?

```suggestion
    if (["autoTestPCR", "imageRight", "rules"].includes(document)) {
```